code/style.css

```diff
@@ -1,5 +1,371 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+@import url('https://fonts.googleapis.com/css2?family=Bungee+Shade&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap');
+
```

Comment:
I'm not sure what it is that you are not happy about regarding the footer - but one advice is to add this:
```
body {
  margin: 0;
}
```
to make your header and footer go all the way to the edges (and the footer all the way to the bottom as well) :)